retis-events/src/ovs.rs

```diff
@@ -12,18 +12,21 @@ use crate::{event_section, event_type};
 pub struct OvsEvent {
     /// Event data
     #[serde(flatten)]
-    pub event: OvsEventType,
+    pub event: Option<OvsEventType>,
 }
 
 impl EventFmt for OvsEvent {
     fn event_fmt(&self, f: &mut fmt::Formatter, format: DisplayFormat) -> fmt::Result {
-        self.event.event_fmt(f, format)
+        if let Some(event) = &self.event {
+            return event.event_fmt(f, format);
+        }
+        write!(f, "?")
```

Comment:
Probably not

Reply:
Although... not sure if returning an error here is the correct thing to do. According to the documentation, `fmt::Error` should only be raised in limited situations:
https://doc.rust-lang.org/stable/core/fmt/struct.Error.html
Maybe instead of "?"  it can return "Error: OVS event missing"?